playbooks/templates/metadata_pod.yml.j2

```diff
@@ -0,0 +1,43 @@
+{% if metadata.collection is sameas true and metadata.targeted is sameas true %}
```

Comment:
@rsevilla87 how is this any different from playbooks/templates/metadata.yml.j2 ? A side by side looks the same at a glance.

Reply:
indentation ;)

Reply:
Ahhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh. Gothca thanks :-)